### PR TITLE
Added ssh, and fixed syntax.

### PR DESCRIPTION
**Pull request by ChrisForti**
The vm launch is a little janky, so maybe someone else can test it.

### CODE REVIEW

File: webserver-darwin.sh [hunk 1]
```diff
@@ -1,36 +1,54 @@
...
 else
   echo "installing nano"
   brew install nano 
 fi
- echo "installing multipass on $(uname)"
-  if [ "$(uname)" = "Darwin" ]
+
+# Install multipass
+  echo "installing multipass on $(uname)"
+if [ "$(uname)" = "Darwin" ]
```
Comment (stephenlauck) [inline webserver-darwin.sh:35]: Why are we checking "if darwin" if the the script is called webserver-darwin.sh and if we have good instructions in the README.md?

File: webserver-darwin.sh [hunk 2]
```diff
@@ -1,36 +1,54 @@
...
 
-echo "brew should be installed"
+# Install homebrew
+  echo "brew should be installed"
 if ( which brew )
 then
   echo "brew already installed"
 else
   echo "installing brew"
   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```
Comment (stephenlauck) [inline webserver-darwin.sh:10]: Is this non-interactive when it runs? Is there a way to make this non-interactive?

**Review by stephenlauck — CHANGES_REQUESTED** [review]

File: webserver-darwin.sh (hunk 2 above)
Comment (ChrisForti) [reply webserver-darwin.sh:10]: No it is not. yes i believe so, i will work on that.

File: webserver-darwin.sh (hunk 1 above)
Comment (ChrisForti) [reply webserver-darwin.sh:35]: Are you saying I shouldn't check?

**Review by stephenlauck — APPROVED** [review]
let's fast follow with the interactive fix.

**Pull request merged**